fov_processing_pipeline/wrappers.py

```diff
@@ -9,6 +9,65 @@
 from . import data, utils, stats, reports, postprocess
 
 
+@task
+def save_load_data(save_dir, protein_list=None, n_fovs=100, overwrite=False, dataset="quilt"):
+    """
+    Retreives or loads data.
+
+    Parameters
+    ----------
+    save_dir: str
+        save directory of results
+
+    trim_data: bool or int
+        do we trim the data or not, or how many data to we trim the dataset to
+
+    overwrite: bool
+        do we overwrite the files if they exist? (i.e. do you want to put new results in an old directory)
+
+    dataset: str
+        can be "quilt" or "labkey"
+
+    Returns
+    -------
+    cell_data: pandas.DataFrame
+        Dataframe where each row corresponds to a single cell
+
+    fov_data: pandas.DataFrame
+        Dataframe where each row corresponds to an FOV
+
+    """
+
+    cell_data_path = "{}/cell_data.csv".format(save_dir)
+    fov_data_path = "{}/fov_data.csv".format(save_dir)
+
+    if not os.path.exists(cell_data_path) or overwrite:
+
+        import pdb
+        pdb.set_trace
```

Comment:
Remove trace code

Reply:
oops, yeah, I need to have better tests to catch that

Reply:
I am wondering if there is like a `flake8` look for `pdb`

Reply:
Or just `flake8` look for specific string

Reply:
https://pypi.org/project/flake8-debugger/